.github/workflows/octokit.yml

```diff
@@ -26,18 +26,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu ]
-        ruby: [ 2.3, 2.4, 2.5, 2.6, 2.7, '3.0', '3.1', head ]
+        ruby: [ 2.7, '3.0', '3.1', head ]
         faraday: [ '~> 1.0', '~> 2.0' ]
```

Comment:
I am pretty sure we can get rid of the 1.0 reference here... Let me know if that sounds right.

```suggestion
        faraday: '~> 2.0'
```

Reply:
We can still continue to support Faraday 1.x. Dropping support for Faraday 1.x is often pretty painful as there are lots of packages out there that still require it.

Reply:
> Den-bkash

<img width="512" alt="debug_concurrency_issues_tutorial_procedure_4" src="https://user-images.githubusercontent.com/110720814/186065411-eda3fef6-f32c-498c-b6e7-c524f7225018.png">

Reply:
> I am pretty sure we can get rid of the 1.0 reference here... Let me know if that sounds right.
> 
> ```suggestion
>         faraday: '~> 2.0'
> ```